src/costs.jl

```diff
@@ -6,7 +6,15 @@ else
 end
 
 
+"""
+    Instruction
 
+`Instruction` is like a `GlobalRef` except it isn't. (Is this because you need it to be
+encodable in a type parameter? How do you handle submodules? Would making `mod` the result
+of `fullname` fix that?)
+
+Also, are these just low-level CPU instructions or is `Base.sin` allowed?
```

Comment:
`sin` is included in the `COST` dict. `Base.sin(::SVec)` dispatches to a SIMD implementation.
If the user is on Linux with a recent version of GLIBC, it will use the method in `libmvec.so.1`. Otherwise, it'll use the implementation defined in `SLEEFPirates.jl`.

These represent function calls, which can be expensive, like [clenshaw](https://github.com/chriselrod/LoopVectorization.jl/blob/master/test/runtests.jl#L5) called in the [clenshaw!](https://github.com/chriselrod/LoopVectorization.jl/blob/master/test/miscellaneous.jl#L205) test.